### PR TITLE
Issue#4

### PR DESCRIPTION
**Pull request by LorenzoEscriva**
add JournalEntry class and test.

### CODE REVIEW

File: AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/IllegalJournalEntryException.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.tap.project.escrivaghera.AccountantApp;
+
+public class IllegalJournalEntryException extends Exception {
+
+	public IllegalJournalEntryException(String error) {
```
Comment (matteoghera) [inline AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/IllegalJournalEntryException.java:5]: The constructor must be empty, create a final static field with following declaration: "The total of counts is several"

Comment (LorenzoEscriva) [reply AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/IllegalJournalEntryException.java:5]: the constructor is empty....the string was passed from the class who call the exception. Look at the class who call that exception...

File: AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/JournalEntry.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		listOfCount = new ArrayList<Count>();
+	}
+
+	public List<Count> getListOfCount() {
+		return listOfCount;
+	}
+
+	public void setListOfCount(List<Count> list) throws IllegalJournalEntryException {
+		int countleft = 0, countright = 0;
+		for (int i = 0; i < list.size(); i++) {
```
Comment (matteoghera) [inline AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/JournalEntry.java:26]: Sum of value in the Count class

File: AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/JournalEntry.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+			if (list.get(i).isLeft == true) {
+				countleft++;
+			} else {
+				countright++;
+			}
+		}
+		if (countleft != countright)
+			throw new IllegalJournalEntryException("The total of counts is several");
+		this.listOfCount = list;
```
Comment (matteoghera) [inline AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/JournalEntry.java:37]: add the difference between countleft and countright

File: AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/IllegalJournalEntryException.java (hunk 1 above)
Comment (matteoghera) [reply AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/IllegalJournalEntryException.java:5]: ok

File: AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/JournalEntry.java (hunk 2 above)
Comment (matteoghera) [reply AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/JournalEntry.java:26]: Remember the test

Comment (LorenzoEscriva) [reply AccountantApp/src/main/java/com/tap/project/escrivaghera/AccountantApp/JournalEntry.java:26]: Yes, I miss understood. I will change soon.

**Pull request merged**